src/redisearch_api.c

```diff
@@ -39,6 +39,7 @@ IndexSpec* RS_CreateIndex(const char* name, RSGetValueCallback getValue, void* g
   spec->maxPrefixExpansions = -1;
   spec->getValue = getValue;
   spec->getValueCtx = getValueCtx;
+  DocTable_EnableIdArray(&spec->docs);
```

Comment:
we need to use DocTable_IdArrayGet on iterator next function on the llapi and also use it on criteria tester